test/test_transmissions.py

```diff
@@ -2,6 +2,8 @@
 import json
 import os
 import tempfile
+import warnings
+from mock import patch
```

Comment:
Move this below the external libraries. The ordering should follow this format:

1. Standard library (built-in) `import`s
2. Standard library (built-in) `from`s

line break

3. External library `import`s
4. External library `from`s

line break

5. Local `import`s
6. Local `from`s

In each the imports/froms should be in alphabetical order.